test/WebSites/ApiExplorerWebSite/Controllers/ApiExplorerResponseContentTypeController.cs

```diff
@@ -15,20 +15,6 @@ public Product Unset()
         }
 
         [HttpGet]
-        [Produces("*/*")]
-        public Product AllTypes()
-        {
-            return null;
-        }
-
-        [HttpGet]
-        [Produces("text/*")]
-        public Product Range()
-        {
-            return null;
-        }
-
-        [HttpGet]
         [Produces("application/json")]
```

Comment:
Are there examples where an action has multiple content-types supported? Looking at the updated version of this controller it's a bit too simple